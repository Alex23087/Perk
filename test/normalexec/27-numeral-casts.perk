fun main () : int {
    let x := 5;
    let y := 5 + .5;
}