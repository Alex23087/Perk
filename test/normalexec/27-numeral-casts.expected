x: 5, y: 5.50
